### PR TITLE
Pre-populate test/test-helper.el and test/NAME-test.el

### PR DESCRIPTION
**Pull request by tarsius**
This follows #38, in particular https://github.com/rejeep/ert-runner.el/issues/38#issuecomment-282552310 ff.

I am only adding basic header and footer lines. It might however be a good idea to insert some templates in between to make it easier for users to get started. I'll leave that to you because I don't know what conventions you would prefer users to follow here. If you do that, then feel free to attribute the amended/new commit to yourself instead of to me.

We could also add a comment like `;; This isn't a library, so no feature should be provided`, to make it even more likely that users won't do that. But that seems like a bit much.

### CODE REVIEW

File: ert-runner.el [hunk 1]
```diff
@@ -210,12 +210,23 @@ nil, `ert-runner-test-path' will be used instead."
...
-  (f-touch (f-join ert-runner-test-path "test-helper.el"))
-  (f-touch (f-join ert-runner-test-path (s-concat name "-test.el")))
   (message "create %s" (ansi-green (f-filename ert-runner-test-path)))
+  (f-mkdir ert-runner-test-path)
   (message "create  %s" (ansi-green "test-helper.el"))
-  (message "create  %s" (ansi-green (s-concat name "-test.el"))))
+  (let ((test-file (s-concat name "-test.el")))
+    (with-temp-file (f-join ert-runner-test-path "test-helper.el")
+      (insert (format "\
+;;; test-helper.el --- Helpers for %s
```
Comment (phst) [inline ert-runner.el]: I think this should enable lexical binding by appending `-*- lexical-binding: t; -*-` to the first line. Same below.

Comment (tarsius) [inline ert-runner.el]: That might be nicer, but it's not essential. And I don't want to add anything to this pr that might delay it being merged.

**Comment by tarsius** [comment]
@rejeep Could you please have a look at this?

**Comment by tarsius** [comment]
@rejeep Could you please merge this?

As long as you don't I have to keep opening pull requests for packages that use this tool.

**Comment by tarsius** [comment]
Thanks again!

**Pull request merged**